Update/omot_021.txt

```diff
@@ -2993,7 +2993,7 @@ void main()
 			NULL, "\"I think the possibility that he was killed is low.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 3, "ps3/s14/03/220300098", 256, TRUE);
 	OutputLine(NULL, "もしも死んでいたら、鉄平じゃなくて富竹さんが祟りの犠牲として発見されていたはずだからね」",
-			NULL, " If Tomitake-san were dead, he would've been discovered as the victim of the curse, instead of Teppei.\"", Line_Normal);
+			NULL, " Because If Tomitake-san were dead, he would've been discovered as the victim of the curse, instead of Teppei.\"", Line_Normal);
```

Comment:
```suggestion
			NULL, " Because if Tomitake-san were dead, he would've been discovered as the victim of the curse, instead of Teppei.\"", Line_Normal);
```